src/Home.jsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  color: #383854;
+  font-family: Righteous;
+  margin: 2rem 0 0 0;
+  font-size: 2rem;
+`;
+
+const Title = styled.h1`
+  font-size: 3rem;
+  width: 100%;
+  background-color: rgba(0, 0, 0, 0.25);
+  border: 0.05rem rgba(0, 0, 0, 0.5) solid;
+  margin: 0 0 0.25rem 0;
+  padding: 0.25rem;
+  @media only screen and (max-width: 700px) {
```

Comment:
Love you are styling for mobile, I think it would be good to check it on your phone when you are styling it not just look on the computer. I think some things are very small on the phone screen.